### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by melinath**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+* compute: added `disk.resource_policies` field to resource `google_compute_instance_template` ([#2929](https://github.com/hashicorp/terraform-provider-google-beta/pull/2929))
+* provider: added plan time validations for fields that expect base64 values. ([#2906](https://github.com/hashicorp/terraform-provider-google-beta/pull/2906))
+* sql: added support for point-in-time-recovery to `google_sql_database_instance` ([#2923](https://github.com/hashicorp/terraform-provider-google-beta/pull/2923))
+* monitoring : added `availability` sli metric support for the resource `google_monitoring_slo` ([#2908](https://github.com/hashicorp/terraform-provider-google-beta/pull/2908))
+
+BUG FIXES:
+* bigquery: fixed bug where you could not reorder columns on `schema` for resource `google_bigquery_table` ([#2913](https://github.com/hashicorp/terraform-provider-google-beta/pull/2913))
+* cloudrun: supressed `run.googleapis.com/ingress-status` annotation in `google_cloud_run_service` ([#2920](https://github.com/hashicorp/terraform-provider-google-beta/pull/2920))
```
Comment (slevenick) [inline CHANGELOG.md:14]:
```suggestion
* cloudrun: suppressed `run.googleapis.com/ingress-status` annotation in `google_cloud_run_service` ([#2920](https://github.com/hashicorp/terraform-provider-google-beta/pull/2920))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,18 @@
 ## 3.56.0 (Unreleased)
+FEATURES:
+* **New Resource:** `google_privateca_certificate` ([#2924](https://github.com/hashicorp/terraform-provider-google-beta/pull/2924))
+
+IMPROVEMENTS:
+* bigquery: added `status` field to `google_bigquery_job` ([#2926](https://github.com/hashicorp/terraform-provider-google-beta/pull/2926))
+* compute: added `disk.resource_policies` field to resource `google_compute_instance_template` ([#2929](https://github.com/hashicorp/terraform-provider-google-beta/pull/2929))
+* provider: added plan time validations for fields that expect base64 values. ([#2906](https://github.com/hashicorp/terraform-provider-google-beta/pull/2906))
```
Comment (slevenick) [inline CHANGELOG.md:8]: I think this should be `* all:` instead of provider

Comment (melinath) [reply CHANGELOG.md]: Fixed. It looks like both are used - what makes this one all instead of provider?

Comment (slevenick) [reply CHANGELOG.md]: Hm, I thought we just used `all:`. No preference then if we have both

**Review by slevenick — APPROVED** [review]

**Pull request merged**